roaring/roaring_internal_test.go

```diff
@@ -1869,6 +1869,38 @@ func TestXorRunRun(t *testing.T) {
 	}
 }
 
+func TestBitmapFlip(t *testing.T) {
```

Comment:
this test is really confusing - it looks like a table driven test, but it's really only testing a single bitmap flip. We should probably rewrite this so that it follows the same pattern as the other table driven tests in this file. Using bitmapOdds and bitmapEvens helper funcitons (which may not be available in this PR) would be a perfect fit.

Reply:
Good call. I'll open an issue to re-write this test using those helper functions.